src/Dialog/Dialog.js

```diff
@@ -141,7 +141,7 @@ function getStyles(props, context) {
       fontSize: dialog.bodyFontSize,
       color: dialog.bodyColor,
       padding: gutter,
-      paddingTop: props.title ? 0 : gutter,
+      paddingTop: 0,
```

Comment:
I was getting a unnecessary padding in the top of the dialog with the current value in the HEAD.

`paddingTop: props.title ? 0 : gutter,`

I checked the other dialogs with paddingTop : 0 and it wasn't breaking anything. I saw you made some changes again on this.